uv.lock

```diff
@@ -0,0 +1,370 @@
+version = 1
```

Comment:
please don't add uv.lock - at least not in this PR

Reply:
removed :ballot_box_with_check: